doc/nrf/releases_and_maturity/release_notes.rst

```diff
@@ -10,15 +10,17 @@ This page is included only in the latest documentation, because it might contain
 
 .. note::
    A "99" at the end of the version number of this documentation indicates continuous updates on the main branch since the previous major.minor release.
-   When looking at this latest documentation, be aware of the following aspect:
+   When looking at this latest documentation, be aware of the following aspects:
 
+   * Changes between releases are tracked on the :ref:`ncs_release_notes_changelog` page, but the main branch might contain additional changes that are not yet listed on that page.
    * The release note pages that are available in the latest documentation might differ slightly from the release notes that were included in the respective |NCS| release at its release date.
      Therefore, to see the official version of the release notes for a specific |NCS| release, switch to the documentation for the corresponding |NCS| version using the selector in the upper left corner.
 
 .. toctree::
    :maxdepth: 1
    :caption: Subpages:
 
+   releases/release-notes-changelog
```

Comment:
I think we should also update the doc/version.json file to be in sync with the version.json file in main, which also has the 
"3.0.0-preview1" preview release.

Reply:
No, this would be incorrect. This branch is independent, just some cherrypicked commits will go on main. This can stay as is as future -1 releases will still come from this branch.

Reply:
But, during the nRF54H20-1 release, when the version.json file will be updated with the nRF54H20-1 version in main (https://github.com/nrfconnect/sdk-nrf/pull/20674/files#diff-1a65aab35f3d400796ce1241098291e62a8afcc2454f6bf9a22e55d5c8aaa919) and backported to this branch, the version.json file in the branch needs to be in line with the main (which also has the 3.0.0-preview1 version), or will you be fixing the conflicts that time? Both ways would work.